src/supported-keyframes.js

```diff
@@ -0,0 +1,14 @@
+import prefix from './prefix'
+
+/**
+ * Test if a keyframe at-rule should be prefixed or not
+ *
+ * @param {String} vendor prefix string for the current browser.
+ * @return {Boolean}
```

Comment:
Boolean|string - returns `false` or @keyframes identifier with supported prefix

Reply:
@kof thanks, forgot to change description after last changes